products/jbrowse-web/src/rootModel.ts

```diff
@@ -343,6 +350,59 @@ export default function RootModel(
                 session.setDefaultSession()
               },
             },
+            {
+              label: 'Import session…',
+              icon: PublishIcon,
+              onClick: (session: SessionWithWidgets) => {
+                const widget = session.addWidget(
+                  'ImportSessionWidget',
+                  'importSessionWidget',
+                )
+                session.showWidget(widget)
+              },
+            },
+            {
+              label: 'Export session',
+              icon: GetAppIcon,
+              onClick: (session: IAnyStateTreeNode) => {
+                const sessionBlob = new Blob(
+                  [JSON.stringify({ session: getSnapshot(session) }, null, 2)],
+                  { type: 'text/plain;charset=utf-8' },
+                )
+                saveAs(sessionBlob, 'session.json')
+              },
+            },
+            {
+              label: 'Open session…',
+              icon: FolderOpenIcon,
+              onClick: (session: SessionWithWidgets) => {
+                const widget = session.addWidget(
+                  'SessionManager',
+                  'sessionManager',
+                )
+                session.showWidget(widget)
+              },
+            },
+            {
+              label: 'Save session',
+              icon: SaveIcon,
+              onClick: (session: SessionWithWidgets) => {
+                // @ts-ignore
+                if (session.saveSessionToLocalStorage) {
+                  // @ts-ignore
+                  session.saveSessionToLocalStorage()
+                  // @ts-ignore
```

Comment:
any way we can do better with types here?

Reply:
ts-ignores are unneeded now, can access the method directly on jbrowse-web rootModel instead of thru sessionModel